docs/accordproject-concepts.md

```diff
@@ -5,31 +5,31 @@ title: Key Concepts
 
 ## What is an Accord Project Template?
 
-An Accord Project template ties legal text (for a clause or contract) to computer code. It is composed of three elements: 
+An Accord Project template ties legal text (for a clause or contract) to computer code. It is composed of three elements:
 
 - **The Template Text**: the natural language of the template
-- **The Template Model**: the data model that backs the template 
+- **The Template Model**: the data model that backs the template
 - **The Template Logic**: the executable business logic for the template
 
 ![Accord Project Template](assets/020/template.png)
 
-When combined these three elements allow templates to be edited, validated, verified, and then executed on any computer platform (on your own machine, on a Cloud platform, on Blockchain, etc).
+The three components (Text - Model - Logic) can also be intuitively understood as a **progression**, from _human-readable_ legal text to _machine-readable_ code. When combined these three elements allow templates to be edited, validated, verified, and then executed on any computer platform (on your own machine, on a Cloud platform, on Blockchain, etc).
 
-> Note that we use the computing term 'execution' here, which means run by a computer. This is distinct from the legal term 'execution' which usually refers to the process of signing an agreement.
+> Note that we use the computing term 'execution' here, which means run by a computer. This is distinct from the legal term 'execution', which usually refers to the process of signing an agreement.
 
 ### Cicero
 
-The main software implementation for the Accord Project is called [Cicero](https://github.com/accordproject/cicero). It allows users of the accord project to create and manipulate software templates which tie together all three components in the triangle above (i.e., the text, model and logic).
+The main software implementation for the Accord Project is called [Cicero](https://github.com/accordproject/cicero). It allows users of the Accord Project to create and manipulate software templates which tie together all three components in the triangle above (i.e., the text, model and logic).
 
-_More information about how to install Cicero and get started with Accord project templates can be found in the [Installation](accordproject-installation) Section of this documentation._
+_More information about how to install Cicero and get started with Accord Project templates can be found in the [Installation](accordproject-installation) Section of this documentation._
```

Comment:
I'm pretty sure this link is wrong and should be `started-installation`. Would be worth checking that all the links work.